docs/HowTo/Monitor-Nodes/Metrics.md

```diff
@@ -0,0 +1,86 @@
+---
+description: Monitoring and metrics
+---
+
+# Use metrics to monitor performance
+
+Use the [`--metrics-enabled`](../../Reference/CLI/CLI-Syntax.md#metrics-enabled) option to enable
+the [Prometheus](https://prometheus.io/) monitoring and alerting service to access EthSigner.
+
+## Install Prometheus
+
+To use Prometheus with EthSigner, install the
+[Prometheus main component](https://prometheus.io/download/). On MacOS, install with
+[Homebrew](https://formulae.brew.sh/formula/prometheus):
+
+```bash
+ brew install prometheus
+```
+
+## Setting up and running Prometheus with EthSigner
+
+To configure Prometheus to run with EthSigner:
+
+1. Configure Prometheus to poll EthSigner. For example, add the following YAML fragment to the
+   the `prometheus.yml` file:
+
+    !!! example "Example configuration"
+
+        ```yml
+        global:
+          scrape_interval: 15s
+        scrape_configs:
+          - job_name: "prometheus"
+            static_configs:
+            - targets: ["localhost:9090"]
+          - job_name: "ethsigner-prod"
+            scrape_timeout: 10s
+            metrics_path: /metrics
+            scheme: http
+            static_configs:
+            - targets: ["localhost:8546"]
+        ```
+
+    !!! note
+    
+        Alternatively create your own YAML file and add the above example configuration.
+
+1. [Start Besu].
```

Comment:
nit: Not sure we need to say start Besu here. Besu is normally used with EthSigner but doesn't have to be.